DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Controllers/NoteModifyController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Notes.WebApi.Services;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi.Controllers
+{
+    public class NoteModifyController : ApiController
```

Comment:
Created NoteQueries and NoteCommands to handle work with context. Changes commited.